src/index.js

```diff
@@ -69,6 +69,7 @@ export FacetSearchBar from './components/FacetSearchBar';
 export FormattedDate from './components/FormattedDate';
 export GridLayout from './components/GridLayout';
 export Header360 from './components/Header360';
+export IfAllowed from './components/IfAllowed';
```

Comment:
You'll need to add it to the style guide config and add an example file for it there, too, to get it to build. Also, it's a good way to validate it without having to actually use it in an application. You can probably look for one of the other example markdown files that uses the AuthUtils class to set stuff up since you'll need to fake a logged-in user to do this.

Reply:
Okay, I'll look into that